brave/browser-laptop
# Improve UX of unavailable ledger balance

Fix #3778 #3785

Adds `ledgerInfo.error` for debugging and UI.

Auditors: @mrose17 @bbondy 

Test Plan:

Offline case:
1. Disable internet
2. Re/open Brave
3. Open Preferences -> Payments
4. Observe that account balance shows "error, can't retrieve data" and not 0

Online case:
1. In Brave, open Preferences -> Payments and leave it open
2. Close and reopen Brave
4. Observe Payments balance shows "loading" or the correct amount

Offline case / Expected result:
![screen shot 2016-09-07 at 18 54 33](https://cloud.githubusercontent.com/assets/521861/18334505/f067be6e-752c-11e6-8e36-c4b71cc099ca.png)
